.github/workflows/neuron-ci.yml

```diff
@@ -237,11 +237,9 @@ jobs:
                                 -Wno-unused-function \
                                 -Wno-unused-but-set-variable \
                                 -Wno-reorder \
-                                -Wno-sign-compare \
-                                -Wno-strict-aliasing \
-                                -Wno-missing-braces \
-                                -Wno-mismatched-tags" \
-                             -DNRN_EXTRA_MECH_CXX_FLAGS="")
+                                -Wno-sign-compare" \
+                             -DNRN_EXTRA_MECH_CXX_FLAGS="-Wno-sometimes-uninitialized \
+                                -Wno-missing-braces")
```

Comment:
Suggest moving this back to `NRN_EXTRA_CXX_FLAGS` (the CI on this PR is flagging violations, and it doesn't seem worth backporting the fixes)